tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -13,10 +13,10 @@ define BUILD
 	$(MK_DIR)/kata-deploy-binaries-in-docker.sh $(if $(V),,-s) --build=$1
 endef
 
-kata-tarball: | all-parallel merge-builds
+kata-tarball: | make-builds merge-builds
```

Comment:
> Considering the results of the measurement, please, don't drop it.
> 
> I'm fine about making it non default, as long as we change then our GitHub actions to still use the parallel builds there. Does this sound a reasonable compromise?

The idea behind this change is to simplify the `make kata-tarball` flow so that we can fix other issues spotted by @dgibson, in order to make it usable by developers. We certainly don't want `make kata-tarball` to have a different behaviour locally compared to GH at the end of the day. I'll just add the WIP label and dig some more.

Reply:
Perfect, thanks!